Standards/CCSMB-14.md

```diff
@@ -0,0 +1,56 @@
+# *CCSMB 14:* Stereo television
+
+*Author: @notbronwyn*  
+*Version: v1.0*  
+*Last updated: 2024-03-30*  
+
+## Server:
+Videos should either be in BIMG but other formats do not break compliance as long as they are marked in "type" to avoid issues with clients and there is an open-source encoder and decoder
+  
+  an example of this would be [PBB](https://raw.githubusercontent.com/craftytv/pigutv/main/pbbconverter.py) which is supported in the example server
+
+Broadcasts in BIMG should only use the default CraftOS pallete
+  
+Each frame of video should have enough audio to last the entire frame and no more
+
+Each packet sent from the server most have "protocol" set to "stereovideo"
+
+Each packet should have metadata for the TV broadcast
+
+An optional feature that packets can have is subtitles
+
+An example of a compliant server is [here](https://github.com/craftytv/television/tree/main/channel)
+
+Example of a packet:
+```lua
+{
+  protocol = "stereovideo", -- Always stereo video
+  type = broadcastType, -- Image format (BIMG, etc.)   SHOULD BE ALL LOWERCASE
+  audio = {
```

Comment:
If we want to create "similar" protocols, consider looking at CCSMB-5's way of implementing channels,
audio[1] would be left,
audio[2] would be right,
audio[3-8] would be self assigned